### DISCUSSION TITLE
How can I delete the connect record of tabby？

### DISCUSSION THREAD

**Discussion by pooneyy** [Eugeny/tabby#9581] [Q&A] [answered]
As the title suggests.
![image](https://github.com/Eugeny/tabby/assets/85266337/cf493b0e-72f9-4c30-aa2a-11277a6cdb01)

**Comment by moemoechu** [comment, marked as answer]
![PixPin_2024-04-15_11-10-47](https://github.com/Eugeny/tabby/assets/6707029/d27fa5d5-ca3d-40f1-88e9-abe95064682c)

here.

**Reply by pooneyy** [reply]
thanks
